manifests/master/simp_auth.pp

```diff
@@ -1,55 +1,48 @@
 # Add SIMP-specific entries to PuppetServer's auth.conf
 #
+# For documentation about *_allow and *_deny, see the puppetserver docs
+# @see https://puppet.com/docs/puppetserver/5.2/config_file_auth.html#allow-allow-unauthenticated-and-deny
+#
+# @param server_distribution Puppet open source or PE
+#
 # @param auth_conf_path
-#   Type:    Stdlib::AbsolutePath
-#   Default: /etc/puppetlabs/puppetserver/conf.d/auth.conf
 #   The location to the puppet master's auth.conf
 #
 # @param legacy_cacerts_all
```

Comment:
Why are we supporting legacy pki?

Reply:
I think now is a good time to have this discussion

Reply:
This can almost definitely be booted but we can't drop the parameter. I'm thinking that we should just add a deprecation warning if someone uses it and drop the legacy PKI code otherwise.